tests/unit/inference/test_inference.py

```diff
@@ -557,3 +557,53 @@ def test(self, model_family, model_name, task):
                        ds_output["results"][task]["ppl"])
         #assert ds_time <= bs_time
         assert ppl_diff < 0.01
+
+
+@pytest.mark.inference
+@pytest.mark.parametrize("model_w_task",
+                         [("bigscience/bloom-560m",
```

Comment:
We don't currently have CG support for generative models, can we switch this to a BERT variant?

Reply:
yup, I noticed this in the inference test errors. I'll make that change.